### PR TITLE
Added Tests for generator page.

### LINKED ISSUE #1568: Write Selenium tests for generator page.
**Actual Behaviour**

Recently we got a PR merged which disrupted generator page in terms of functionality.

**Expected Behaviour**

Writing unit tests would give a way to test the PRs.

**Would you like to work on the issue?**
Yes.

### PR DESCRIPTION
**Pull request by agbilotia1998**
Partially Fixes #1568

### CODE REVIEW

**Comment by agbilotia1998** [comment]
@geekyd @Princu7 please check these tests.

**Comment by agbilotia1998** [comment]
@geekyd please check I have added some more tests.

File: src/www/index.html [hunk 1]
```diff
@@ -148,149 +148,180 @@
...
 
 <body>
-  <div class="custom-navbar">
-    <a href='.' class="custom-navtitle">
-      <strong>Open Event Webapp Generator</strong>
-    </a>
-    <div class="custom-menubutton">
-      <i class="glyphicon glyphicon-th"></i>
-    </div>
+<div class="custom-navbar">
```
Comment (geekyd) [inline src/www/index.html]: Please do not change the indentation, this is incorrect indentation.

Comment (agbilotia1998) [reply src/www/index.html]: Oh,I was testing some changes and finally ran code reformatter with 2 indent spaces.Can you please once have a look ?

**Comment by agbilotia1998** [comment]
@geekyd done please check!

File: src/www/index.html [hunk 2]
```diff
@@ -253,8 +253,10 @@
         </ul><label class="white-color">Choose your data source</label>
         <ul style="list-style-type:none">
           <!--<li><input type="radio" name="datasource" value="mockjson"> Default mock API </input></li>-->
-          <li id="uploadJSON"><input name="datasource" type="radio" value="jsonupload" class="white-color"> Upload your own JSON files</li>
-          <li id="endpointAPI"><input name="datasource" type="radio" value="eventapi" class="white-color"> API endpoint of event on OpenEvent</li>
+            <li id="uploadJSON"><input name="datasource" id="jsonupload" type="radio" value="jsonupload"> Upload your own JSON files
```
Comment (geekyd) [inline src/www/index.html]: Please fix indentation.

**Comment by agbilotia1998** [comment]
@geekyd done please review!

**Comment by agbilotia1998** [comment]
@geekyd please run a travis build again here.

**Comment by agbilotia1998** [comment]
@geekyd please review!!

**Pull request merged**